modules/claims_api/app/models/claims_api/concerns/file_data_validation.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module FileDataValidation
```

Comment:
this doesn't actually validate anything so maybe a different name?

Reply:
Just calling it FileData then...